File: parse_vlan_txt.py
#!/use/bin/python

#Python has a handy external library available to read CSV files
import csv

#open the CSV File for reading

with open('vlan.txt') as input_file:
        line_reader = csv.reader(input_file, delimiter='\t')
#delimiter of line is set to tab character represented as \t
        for row in line_reader:
                ports = row[3].split(",")
                for port in ports:
                        iosconfig = """\
int g{} 
switchport
switchport mode access
switchport access vlan {}
no shut \n\n\n
                                    """
                                      
                        print iosconfig.format(port,row[1])

File: parse_vlan_txt.js
#!/usr/local/bin/node

// Load external library used open and write to files on disk.
const fs = require('fs');

//Load external library used parse delimited files.
var parse = require('/usr/local/lib/node_modules/csv-parse/lib/sync');

//Open file for reading
var fileDataA = fs.readFileSync('vlan.txt','utf8');
recordsA = parse(fileDataA, {delimiter: "\t"} );

recordsA.forEach(function(row) {
        ports = row[3].split(",");
        ports.forEach(function(port){
                iosconfig = `
int g${ port }
switchport
switchport mode access
switchport access vlan ${ row[1] }
no shut \n\n\n `;
                console.log(iosconfig);
        });
});

